Framework/Sources/UI/Cells/BooleanTweakTableViewCell.swift

```diff
@@ -7,9 +7,8 @@ import UIKit
 
 internal class BooleanTweakTableViewCell: UITableViewCell, TweakViewControllerCell {
 
-    var feature: String?
-    var variable: String?
-
+    private var _feature: String? = nil
```

Comment:
it seems that here and in TextTweakTableViewCell we are regressing the changes of #81 
maybe a wrong `stash apply/pop`?